packages/odata-v4/src/internal.ts

```diff
@@ -10,3 +10,5 @@ export * from './entity';
 export * from './extract-odata-etag';
 export * from './filter-function';
 export * from './filter-functions';
+export * from './common';
+export { DeserializedType } from './de-serializers';
```

Comment:
This line seems to be a left over?

Reply:
I needed to re-export it because both `de-serliazers` and `common` contained `DeserializedType`